resources/translations.ts

```diff
@@ -0,0 +1,150 @@
+import { Lang } from '../types/global';
+import Regexes from './regexes';
+import NetRegexes from './netregexes';
+
+// Fill in LocaleRegex so that things like LocaleRegex.countdownStart.de is a valid regex.
+const localeLines = {
```

Comment:
```suggestion
const localeLines: { [key: string]: Record<Lang, string> } = {
```
I think it's expected that all languages are filled out in this file.  I think especially if we are going to treat these as records, we should enforce that here.  What do you think?

Reply:
True and working for now. But do you remember the time that intl server updated patch 5.3 but CN/KR didn't, we had `en/de/fr/ja` completed but `cn/kr` lacking.
If this happens again, then the type as this would not be correct.

Or, something like this, ensure `en` required only?
```typescript
type LocaleLines = {
  [key: string]: {
    en: string;
  } & Partial<Record<Exclude<Lang, 'en'>, string>>;
};
```

Reply:
Ah, that's true, that maybe they're missing.  But if they're missing then you can't have `Record<Lang, RegExp>` below, because it's not a Record anymore if there are missing keys.

I think there are two alternatives.  Either we make it not a record (like you suggest with Partial above) or we add in something (a fake duplicate regex of what `en` says, so at least it exists even if it's wrong?) for other languages just so something exists.  I don't think we need to list it in localeLines, but you could have the proxy still return something.

I just am worried about having a Record with missing keys.